java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/TraceUtil.java

```diff
@@ -190,20 +173,16 @@ private static void printRequestHeader(Writer out, InputStream stream) {
     }
 
     private static byte printHeader(Writer out, InputStream stream) {
```

Comment:
I reworked the comments in this function to get rid of the commented-out function calls.